### PR TITLE
Add step function config.

### PR DESCRIPTION
**Pull request by MancunianSam**
This has been removed from tdr-terraform-modules under PR
https://github.com/nationalarchives/tdr-terraform-modules/pull/238

The resources haven't changed so this shouldn't need any state migration

### CODE REVIEW

File: templates/step_function/consignment_export_definition.json.tpl [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            "States.Timeout"
+          ],
+          "Next": "Task failed choice"
+        }
+      ],
+      "Parameters": {
+        "LaunchType": "FARGATE",
+        "Cluster": "${cluster_arn}",
+        "TaskDefinition": "${task_arn}",
+        "PlatformVersion": "1.4.0",
```
Comment (TomJKing) [inline templates/step_function/consignment_export_definition.json.tpl:33]: Should the `PlatformVersion` be a parameter?

File: templates/step_function/consignment_export_definition.json.tpl [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      "Resource": "arn:aws:states:::ecs:runTask.waitForTaskToken",
+      "HeartbeatSeconds": 60,
+      "TimeoutSeconds": 1800,
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "States.HeartbeatTimeout",
+            "States.Timeout"
+          ],
+          "MaxAttempts": 3
```
Comment (TomJKing) [inline templates/step_function/consignment_export_definition.json.tpl:16]: Should the `MaxAttempts` be a parameter?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple very minor suggestions, feel free to ignore.

File: templates/step_function/consignment_export_definition.json.tpl (hunk 2 above)
Comment (MancunianSam) [reply templates/step_function/consignment_export_definition.json.tpl]: This one makes sense as a parameter as well, I've changed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**